app/api/discount_codes.py

```diff
@@ -70,6 +73,63 @@ def validate_quantity(self, data, original_data):
                          type_='event')
 
 
+class DiscountCodeSchemaEvent(Schema):
```

Comment:
If both the schemas are almost same then maybe you can try inherit one on other. This may save us from redundancy as well.